source/braille.py

```diff
@@ -2560,10 +2560,11 @@ def _doNewObject(self, regions):
 		self.mainBuffer.update()
 		# Last region should receive focus.
 		self.mainBuffer.focus(region)
-		if isinstance(region, TextInfoRegion):
-			self.scrollToCursorOrSelection(region)
-		elif self.buffer is self.mainBuffer:
-			self.update()
+		if self.buffer is self.mainBuffer:
+			if isinstance(region, TextInfoRegion):

```

Comment:
Why do we only call `scrollToCursorOrSelection` for TextInfoRegion?

Reply:
I am not aware about other regions with cursor or selection. I suppose that `scrollToCursorOrSelection` does nothing if there is no cursor nor selection.
If there is cursor or selection `scrollTo` executes `BrailleHandler.update`.
Regions without cursor or selection needs that `BrailleHandler.update` is executed but if there is cursor or selection `scrollTo` executes it.

Reply:
`scrollToCursorOrSelection` has an explicit check where region is not a `TextInfoRegion`. I guess that code is there for a reason.

Reply:
Thank you for mentioning this. I am sorry. I ignored it because I did not understand it, and things seemed to work. I should have considered it more.
It seems that `not isinstance(region, TextInfoRegion)` might be (is) there for situation that sometimes some other type of regions which are not class of `TextInfoRegion` or its descendants would have selection feature.

Simple solution which does not have issues of #16499 or #16501 would be to go back to code before #16463 except removing line
`		self.updateDisplay()
`
from `BrailleBuffer.scrollTo` function.

Do you have considerations against this?